pipelines/spancat-datasets/README.md

```diff
@@ -0,0 +1,93 @@
+
+
+## 📋 project.yml
+
+The [`project.yml`](project.yml) defines the data assets required by the
+project, as well as the available commands and workflows. For details, see the
+[spaCy projects documentation](https://spacy.io/usage/projects).
+
+### ⏯ Commands
+
+The following commands are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run).
+Commands are only re-run if their inputs have changed.
+
+| Command | Description |
+| --- | --- |
+| `convert-wnut17-ents` | Convert WNUT17 dataset into the spaCy format |
+| `convert-wnut17-spans` | Convert WNUT17 dataset into the spaCy format |
+| `clean-wikineural` | Remove unnecessary indices from wikineural data |
+| `convert-wikineural-spans` | Convert WikiNeural dataset (de, en, es, nl) into the spaCy format |
+| `convert-wikineural-ents` | Convert WikiNeural dataset (de, en, es, nl) into the spaCy format |
+| `clean-conll` | Remove unnecessary indices from ConLL data |
+| `convert-conll-spans` | Convert CoNLL dataset (de, en, es, nl) into the spaCy format |
+| `convert-conll-ents` | Convert CoNLL dataset (de, en, es, nl) into the spaCy format |
+| `convert-archaeo-spans` | Convert Dutch Archaeology dataset into the spaCy format |
+| `convert-archaeo-ents` | Convert Dutch Archaeology dataset into the spaCy format |
+| `convert-anem-spans` | Convert AnEM dataset into the spaCy format |
+| `convert-anem-ents` | Convert AnEM dataset into the spaCy format |
+| `clean` | Remove intermediary files |
+
+### ⏭ Workflows
+
+The following workflows are defined by the project. They
+can be executed using [`spacy project run [name]`](https://spacy.io/api/cli#project-run)
+and will run the specified commands in order. Commands are only re-run if their
+inputs have changed.
+
+| Workflow | Steps |
+| --- | --- |
+| `wnut17` | `convert-wnut17-ents` &rarr; `convert-wnut17-spans` |
+| `wikineural` | `clean-wikineural` &rarr; `convert-wikineural-ents` &rarr; `convert-wikineural-spans` |
+| `conll` | `clean-conll` &rarr; `convert-conll-spans` &rarr; `convert-conll-ents` |
+| `archaeo` | `convert-archaeo-ents` &rarr; `convert-archaeo-spans` |
+| `anem` | `convert-anem-ents` &rarr; `convert-anem-spans` |
+
+### 🗂 Assets
+
+The following assets are defined by the project. They can
+be fetched by running [`spacy project assets`](https://spacy.io/api/cli#project-assets)
+in the project directory.
+
+| File | Source | Description |
+| --- | --- | --- |
+| `assets/wnut17-train.iob` | URL | WNUT17 training dataset for Emerging and Rare Entities Task from Derczynski et al., 2017 |
+| `assets/wnut17-dev.iob` | URL | WNUT17 dev dataset for Emerging and Rare Entities Task from Derczynski et al., 2017 |
+| `assets/wnut17-test.iob` | URL | WNUT17 test dataset for Emerging and Rare Entities Task from Derczynski et al., 2017 |
+| `assets/raw-en-wikineural-train.iob` | URL | WikiNeural (en) training dataset from Tedeschi et al. (EMNLP 2021) |
```

Comment:
Sorry about this, I think this should be [WikiNEuRal](https://github.com/Babelscape/wikineural).